packages/notifications/src/elements/Notification.tsx

```diff
@@ -7,16 +7,31 @@
 
 import React, { HTMLAttributes } from 'react';
 import PropTypes from 'prop-types';
+import InfoStrokeIcon from '@zendeskgarden/svg-icons/src/16/info-stroke.svg';
 import { StyledNotification, IStyledNotificationProps } from '../styled';
+import { StyledIcon } from '../styled/StyledIcon';
 import { ARRAY_VALIDATION_TYPE } from '../utils/types';
+import { validationIcons, validationHues } from '../utils/icons';
 
 /**
  * Supports all `<div>` props
  */
 export const Notification = React.forwardRef<
   HTMLDivElement,
   IStyledNotificationProps & HTMLAttributes<HTMLDivElement>
->((props, ref) => <StyledNotification ref={ref} {...props} />);
+>((props, ref) => {
+  const Icon = props.type ? validationIcons[props.type] : InfoStrokeIcon;
+  const hue = props.type && validationHues[props.type];
+
+  return (
+    <StyledNotification ref={ref} isFloating {...props}>
+      <StyledIcon hue={hue}>
+        <Icon />
+      </StyledIcon>
```

Comment:
Refer to designs; an icon is not a mandatory element for a Notification. When `type === undefined`, there is no icon.